### ISSUE TITLE
arch software

### ISSUE THREAD

**Issue by dotslashtaylor** [dotslashtaylor/dotslashtaylor.github.io#2]
- acpi
- alacritty
- anki
- archey3
- arduino-mk
- audacity
- autoconf
- automake
- base
- binutils
- binutils
- blez-utils
- cheese
- clang
- cmus
- conky
- cups
- efibootmgr
- emacs
- fakeroot
- feh
- file
- findutils
- firefox
- flameshot
- font-manager
- git
- grep
- groff
- grub
- gzip
- htop
- i3-wm
- keepassxc
- libmpdclient
- liferea
- linux
- linux-firmware
- ly
- m4
- make
- man-db
- man-pages
- ncdu
- nerd-fonts-complete
- networkmanager
- os-prober
- pacman 
- patch
- pavucontrol
- pkgconf
- playerctl
- pulseaudio
- python-pip
- ranger
- redshift
- rmlint
- rofi
- sed
- sudo
- system-config-printer
- thefuck
- tldr
- ttf-iosevka
- vi/vim
- wget
- which
- xorg-server
- xorg-xinit
- yay

- borg
- copyq
- croc
- darktable
- dhcpcd
- dmenu
- dunst
- fbreader
- frcon/fronq
- flex
- gawk
- geogebra
- gettext
- gimp
- goldendict
- kicad
- krita
- librecad
- libreoffice-fresh
- libtool
- obs-studio
- openshot
- pcb
- powerline
- python-scipy
- python-sphinx
- qbittorrent
- qcad
- ruby
- termdown
- texinfo
- youtube-dl
